common/pkg/capi/clustertopology/handlers/mutation/meta.go

```diff
@@ -39,10 +41,14 @@ func NewMetaGeneratePatchesHandler(name string, m ...MetaMutater) handlers.Named
 	scheme := runtime.NewScheme()
 	_ = bootstrapv1.AddToScheme(scheme)
 	_ = controlplanev1.AddToScheme(scheme)
+	_ = capav1.AddToScheme(scheme)
+	_ = capdv1.AddToScheme(scheme)
```

Comment:
Good to know

Reply:
Yeah, the meta handler has to be able to decode all kinds. We do still have the fallback to unstructured decoder for any meta handlers we don't have APIs vendored for.